tools/azure-sdk-tools/tests/integration/scenarios/dev_requirement_samples/relative_requirements.txt

```diff
@@ -0,0 +1,13 @@
+tests/testserver_tests/coretestserver
+-e tests/testserver_tests/coretestserver
+-e ../../identity/azure-identity
+../../identity/azure-identity
+-e ../../../tools/azure-devtools
+../../../tools/azure-devtools
+
```

Comment:
Deliberately including this blank line. Because we were accidentally treating the blank line in `eng/test_tools.txt` as a `.` due to path resolution 😆 

That's wrong! We definitely needed to default that.